contributed_definitions/nyaml/NXxrd_pan.yaml

```diff
@@ -136,52 +136,94 @@ NXxrd_pan(NXxrd):
         unit: NX_TIME
         doc: |
           Total time of count.
-    two_theta_plot(NXdata):
+    experiment_result(NXdata):
       doc: |
-        Desired plot would be two_theta vs intensity.
+        All the experiment results data such as scattering angle (2theta),
+        intensity, incident angle, scattering vector, etc will be stored here.
       intensity(NX_FLOAT):
         doc: |
           Number of scattered electrons per unit time.
         dimensions:
           rank: 1
           dim: (nDet,)
       two_theta(NX_FLOAT):
+        unit: NX_ANGLE
         doc: |
-          Axis scale represeting the 2-theta range of the difractogram.
+          The axis scale represents the 2-theta (scattering angle) of the diffractogram.
         dimensions:
           rank: 1
           dim: (nDet,)
       omega(NX_FLOAT):
         exists: optional
+        unit: NX_ANGLE
         doc: |
-          The omega range of the difractogram.
+          The axis scale corresponds to the incident angle of the diffractogram.
         dimensions:
           rank: 1
           dim: (nDet,)
       phi(NX_FLOAT):
         exists: optional
+        unit: NX_ANGLE
         doc: |
-          The phi range of the difractogram
+          The phi range of the diffractogram.
         dimensions:
           rank: 1
           dim: (nDet,)
       chi(NX_FLOAT):
         exists: optional
+        unit: NX_ANGLE
         doc: |
-          The chi range of the difractogram
+          The chi range of the diffractogram
         dimensions:
           rank: 1
           dim: (nDet,)
-    q_plot(NXdata):
+      q_parallel(NX_FLOAT):
+        exists: optional
+        unit: NX_ANY
+        doc: |
+          The scattering vector component is parallel to the sample surface.
+      q_perpendicular(NX_FLOAT):
+        exists: optional
+        unit: NX_ANY
+        doc: |
+          The scattering vector component is perpendicular to the sample surface.
+      q_norm(NX_FLOAT):
+        exists: optional
+        unit: NX_ANY
+        doc: |
+          The concept represents the norm value of the scattering vector, q.
+    q_data(NXdata):
       exists: optional
       doc: |
-        Desired plot would be q_vector vs intensity.
+        The desired plot would be for scattering vectors.
```

Comment:
This does not make sense? I guess it should be `The desired view for scattering vectors`.